src/vs/editor/common/modes.ts

```diff
@@ -1387,6 +1387,7 @@ export interface WorkspaceFileEditOptions {
 	recursive?: boolean;
 	copy?: boolean;
 	folder?: boolean;
+	doNotUseTrash?: boolean;
```

Comment:
Is there reason why the negative prefix is required.  It would be better IMHO to call this useTrash to match the named already used through the code and thus negating the need to flip the setting to match the current variable name?  Example, currently this is a little odd

` { recursive: true, doNotUseTrash: !useTrash }`

and this would be more consistant

 { recursive: true, useTrash: useTrash }

Reply:
Thanks for the suggestion.
The reason is written in my inital coment -> check it out above.

Both names work for me. Let's wait and see what @jrieken prefers

Reply:
The name is a little bulky but I like the motivation. How about `skipTrashBin`?

Reply:
Works for me. Let's go with that name. Thanks